src/redux/actions.ts

```diff
@@ -10,6 +11,7 @@ const actions = {
   EDIT_THING,
   COMPLETE_THING,
   DELETE_THING,
+  SESSION_THING,
```

Comment:
Maybe call this ADD_SESSION? or it is a bit confusing 😆

Reply:
Ok, I thought they should be the same Thing